src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  padding: 0 50px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+
+.nav__list {
+  padding: 0;
+  display: flex;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+
+.header-text {
+  text-decoration: none;
+  position: relative;
+  display: block;
+  align-items: center;
+  line-height: 60px;
+  height: 60px;
+  font-size: 12px;
+
+  font-weight: 500;
+  text-transform: uppercase;
+
+  color: #000;
+}
+
+.is-active {
+  color: #00ACDC;
+}
+
+.header-text:hover {
+  color: #00ACDC;
+}
```

Comment:
The blue color should be stored in the variable as you mentioned in the checklist:
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/f178858a-7847-4a8f-b4de-90aef33b7905)